update-data.rb

```diff
@@ -182,9 +182,14 @@ def main
     puts "images.."
     images()
     puts "images ✅"
+    
+    puts "stories.."
+    # Je ještě potřeba pak předělat appku aby to mělo správně obrázky
```

Comment:
Další commit? :)

Reply:
Jo jo, všiml jsem si toho ještě před založením PR a je to opravený. :-)